prime-dotnet-webapi/Migrations/20200930204650_AgreementVersionType.cs

```diff
@@ -0,0 +1,258 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AgreementVersionType : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Agreement_Enrollee_EnrolleeId",
+                table: "Agreement");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_Agreement_Organization_OrganizationId",
+                table: "Agreement");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_Agreement_Party_PartyId",
+                table: "Agreement");
+
+            migrationBuilder.DropCheckConstraint(
+                name: "CHK_Agreement_OnlyOneForeignKey",
+                table: "Agreement");
+
+            migrationBuilder.CreateCheckConstraint(
```

Comment:
This is in both migrations, which is odd

Reply:
Yeah the first one was mostly had written cause ef doesnt know how to rename things, and im guessing it redid the foreign keys and check constraint because it wanted to do whatever this is:
`migrationBuilder.AlterColumn<int>(
                name: "PartyId",
                table: "Agreement",
                nullable: true,
                oldClrType: typeof(int),
                oldType: "integer");`